File: php/front3.php
<?php
	function front3($string)
	{
		$threechar=substr($string,0,3);
		return $threechar.$threechar.$threechar;
	}
	echo front3("Java")=="JavJavJav" ? "pass" : "fail"; echo "\n";
	echo front3("abc")=="abcabcabc" ? "pass" : "fail"; echo "\n";
	echo front3("Chocolate")=="ChoChoCho" ? "pass" : "fail"; echo "\n";
	echo front3("ab")=="ababab" ? "pass" : "fail"; echo "\n";
?>

File: ruby/front3.rb
def front3(string)
	return string[0...3]*3
end
puts front3("Java")=="JavJavJav" ? "pass" : "fail"
puts front3("abc")=="abcabcabc" ? "pass" : "fail"
puts front3("Chocolate")=="ChoChoCho" ? "pass" : "fail"
puts front3("ab")== "ababab" ? "pass" : "fail"